sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuShuffleEnv.scala

```diff
@@ -89,6 +89,22 @@ object GpuShuffleEnv extends Logging {
     SparkEnv.get.blockManager.externalShuffleServiceEnabled
   }
 
+  // Returns true if authentication is requested, which is not supported
+  // by the RAPIDS Shuffle Manager
+  def isSparkAuthenticateEnabled: Boolean = {
+    val conf = SparkEnv.get.conf
+    conf.getBoolean("spark.authenticate", false)
+  }
+
+  // Returns true if network encryption is enabled in Spark, which
+  // is not handled currently by the RAPIDS Shuffle Manager. Note that
+  // in order to enable network encryption, spark.authenticate must be
+  // already enabled presently.
```

Comment:
Yeah it's a note to self. Basically you have to enable the authentication first, then additionally add wire encryption. So checking for authentication is probably sufficient (right now) but I added the extra check just in case.

I'll rephrase or remove since it is confusing.

Reply:
I agree, spark.authenticate must be enabled so we either don't need this check  or we should expand comment as to why its here.

Reply:
Ok removed!